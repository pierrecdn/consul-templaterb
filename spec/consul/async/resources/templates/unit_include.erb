Myself:
<%= File.basename(template_info['source']) %>, root=<%= File.basename(template_info['source_root']) %>
----
Included File:
<%= render_file('unit_included.erb') %>
----
Myself:
<%= File.basename(template_info['source']) %>, root=<%= File.basename(template_info['source_root']) %>
----