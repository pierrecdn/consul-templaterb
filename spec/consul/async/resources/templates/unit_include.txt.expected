Myself:
unit_include.erb, root=unit_include.erb
----
Included File:
unit_included.erb, root=unit_include.erb
was_rendered_once: true
destination: unit_include.txt
----
Myself:
unit_include.erb, root=unit_include.erb
----